projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployer.java

```diff
@@ -324,10 +325,13 @@ private void updateCronJob(DataJob dataJob, JobDeployment jobDeployment, String
   }
 
   private String getJobVdkImage(JobDeployment jobDeployment) {
-    if (StringUtils.isNotBlank(jobDeployment.getVdkVersion())) {
+    // TODO: Refactor when vdkImage is deprecated.
```

Comment:
nit: same here?

Reply:
Ditto, there is an issue - https://github.com/vmware/versatile-data-kit/issues/1929